src/server/scripts/EasternKingdoms/zone_redridge_mountains.cpp

```diff
@@ -23,16 +23,47 @@ Script Data End */
 
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
+#include "G3DPosition.hpp"
 #include "MotionMaster.h"
+#include "MoveSplineInit.h"
 #include "ObjectAccessor.h"
 #include "WaypointDefines.h"
+#include "World.h"
+#include <algorithm>
 
-enum DumpyKeeshan
+enum RedridgeSpellData
 {
-    NPC_BIGEARL                    = 43248,
+    SPELL_APPLY_QUEST_INVIS_ZONE_3  = 80815, // Used by npc's in Lakeshire Inn
+    SPELL_APPLY_QUEST_INVIS_ZONE_4  = 80816, // Used by npc's in Lakeshire Inn
+
+    SPELL_APPLY_QUEST_INVIS_ZONE_19 = 82099,  // Used by npc's in Lakeshire Townhall
```

Comment:
You can remove those enums, since there unused in your code.
This detect spells needs to be handled in spell_area table.

Reply:
They were moved from other existing code in this file to keep all the spells together. Should I have left them separate?

Reply:
hmm I see. Then keep it for now. We need to remove it after rework of the other script then.